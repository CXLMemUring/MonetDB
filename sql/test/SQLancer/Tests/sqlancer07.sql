START TRANSACTION;
CREATE TABLE "t1" ("c0" DECIMAL(18,3),"c1" BINARY LARGE OBJECT NOT NULL,"c2" DECIMAL(18,3),CONSTRAINT "t1_c1_unique" UNIQUE ("c1"));
PREPARE SELECT DISTINCT (SELECT DISTINCT r'|m<kv' FROM t1 WHERE ((t1.c0)<(?)) GROUP BY t1.c2, ?), ?, t1.c2 FROM t1 WHERE CAST(? AS BOOLEAN) LIMIT 2103332269785059850;
	-- Could not determine type for argument number 2
ROLLBACK;
