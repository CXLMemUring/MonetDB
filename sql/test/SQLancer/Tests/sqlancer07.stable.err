stderr of test 'sqlancer07` in directory 'sql/test/SQLancer` itself:


# 10:16:14 >  
# 10:16:14 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-52758" "--port=31957"
# 10:16:14 >  

MAPI  = (monetdb) /var/tmp/mtest-52758/.s.monetdb.31957
QUERY = PREPARE SELECT DISTINCT (SELECT DISTINCT r'|m<kv' FROM t1 WHERE ((t1.c0)<(?)) GROUP BY t1.c2, ?), ?, t1.c2 FROM t1 WHERE CAST(? AS BOOLEAN) LIMIT 2103332269785059850;
ERROR = !Could not determine type for argument number 2
CODE  = 42000

# 10:16:14 >  
# 10:16:14 >  "Done."
# 10:16:14 >  

