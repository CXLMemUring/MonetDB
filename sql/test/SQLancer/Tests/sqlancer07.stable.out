stdout of test 'sqlancer07` in directory 'sql/test/SQLancer` itself:


# 10:16:14 >  
# 10:16:14 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-52758" "--port=31957"
# 10:16:14 >  

#START TRANSACTION;
#CREATE TABLE "t1" ("c0" DECIMAL(18,3),"c1" BINARY LARGE OBJECT NOT NULL,"c2" DECIMAL(18,3),CONSTRAINT "t1_c1_unique" UNIQUE ("c1"));
#ROLLBACK;

# 10:16:14 >  
# 10:16:14 >  "Done."
# 10:16:14 >  

